.github/workflows/deploy-s3.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           JEKYLL_ENV=production bundle exec jekyll build --strict_front_matter -d _site/training-material
         env:
           SOURCE_TAG: ${{ steps.branch_name.outputs.SOURCE_TAG }}
+          GTN_FORK: ${{ secrets.GTN_FORK }}
```

Comment:
I think this could be `${{ github.repository_owner }}` instead?

Reply:
ah, that would be cleaner, thanks